.eslintrc

```diff
@@ -13,7 +13,7 @@ rules:
   comma-spacing: [2, {before: false, after: true}]
   comma-style: [2, "last"]
   handle-callback-err: [2, "^.*(e|E)rr" ]
-  indent: ["error", 2, {"SwitchCase": 1, "ArrayExpression": "off", "ObjectExpression": "off"}]
+  indent: ["error", 2, {"SwitchCase": 1}]
```

Comment:
Instead of trying to match the lint rules with Prettier, consider using [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier). It will make sure your lint rules don't conflict with prettier. After installing it just add `"extends": ["prettier"]` to the *bottom* of the .eslintrc file.

Reply:
Thanks for pointing it out, that's reasonable. I'm going to update it.